examples/static/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "vip-go-node-skeleton-examples-static",
+  "version": "0.0.1",
+  "description": "Node static service example for WPVIP",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "npx @automattic/vip-go-preflight-checks --verbose --wait=2000",
+    "build": "echo 'No build script step configured, skipping build. For further information, refer to our documentation at https://docs.wpvip.com/technical-references/vip-platform/node-js/#h-requirement-2-production-dependencies-and-npm-scripts'",
+    "start": "node ./src/index.js"
+  },
+  "author": "Roger Theriault",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Automattic/vip-go-node-skeleton/issues"
+  },
+  "homepage": "https://github.com/Automattic/vip-go-node-skeleton/examples/static/",
+  "devDependencies": {},
+  "dependencies": {
+    "express": "^4.17.1",
+    "path": "^0.12.7"
```

Comment:
`path` is a Node.js built-in. No need to install the module (which is just a carbon copy).

Reply:
TIL - thanks!